guides/source/form_helpers.md

```diff
@@ -267,19 +283,18 @@ Outputs:
 </form>
 ```
 
-Note the generated URL contains the `author_id` and `id` delimited by an underscore.
-Once submitted, the controller can [extract each primary key value][] from parameters
-and update the record as it would with a singular primary key.
-
-[extract each primary key value]: action_controller_overview.html#composite-key-parameters
+Note the generated URL contains the `author_id` and `id` delimited by an
+underscore. Once submitted, the controller can [extract each primary key
+value](action_controller_overview.html#composite-key-parameters) from the parameters and update the record as it would with a singular
+primary key.
 
 #### The `fields_for` Helper
 
-The [`fields_for`][] helper creates a similar binding but without rendering a
-`<form>` tag. This can be used to render fields for additional model objects
-within the same form. For example, if you had a `Person` model with an
-associated `ContactDetail` model, you could create a single form for both like
-so:
+The `fields_for` helper is used to render fields for related model objects
+within the same form. The "inner" model is usually related to the "main" model
```

Comment:
Do you think the terms "inner" and "outer" or "main" and "secondary" may be more related?

Reply:
Updated! to a combination of the above :)